flowclient/tests/unit/test_get_status.py

```diff
@@ -4,15 +4,19 @@
 
 from unittest.mock import Mock
 
+import pytest
+
 from flowclient.client import get_status
 
 
-def test_get_status_reports_running():
-    """ Test that status code 202 is interpreted as query running. """
+@pytest.mark.parametrize("running_status", ["queued", "executing"])
+def test_get_status_reports_running(running_status):
+    """ Test that status code 202 is interpreted as query running or queued. """
```

Comment:
Is this test really testing anything? Aren't we just comparing `running_status` to itself because we're setting the return value on the mock to it? Where does the `status_code` come into play at all?